packages/stark-ui/src/modules/message-pane/components/_message-pane.component.scss

```diff
@@ -197,9 +179,23 @@
       }
     }
   }
+}
 
-  & mat-icon {
-    transition: all 0.4s;
+.stark-app-header .stark-message-pane {
```

Comment:
Just double checking... are the changes in this file not breaking anything on the styling of the component? 
Cause I cannot see clearly how this has changed... the diff view is not helping :p

Reply:
Actually, those changes are due to the added stylelint rule `no-descending-specificity: true`
I simply reorganized well the rules 😊 
According to my tests, it does not change anything